### ISSUE TITLE
MSET for multi-datacenter clusters

### ISSUE THREAD

**Issue by Kashif-S** [Netflix/dynomite#771]
It seems that according to #49 mset should be supported for local clusters. From my experience, it doesn't seem to replicate between racks in different data centers. Is there any reason why it was implemented to only support local replication?

Also, it seems that mset is still marked as unsupported in https://github.com/Netflix/dynomite/blame/dev/notes/redis.md

**Comment by ipapapa** [comment]
It is a multi-key command so you will have consistency issues if another client is trying to make a mutation in a remote node at the same time.

**Comment by Kashif-S** [comment]
Why isn't this an issue for replication within local racks?

**Comment by ipapapa** [comment]
Correct, the same issue may appear in a local datacenter but at least you have quorum guarantees on writes within a local Datacenter.
